www/styles/ThemeModal.module.css

```diff
@@ -0,0 +1,24 @@
+.blurBackground {
+  backdrop-filter: blur(3px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+}
+
+.container {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  left: 50%;
+  top: 50%;
+  width: 50em;
+  height: 30em;
```

Comment:
We can let `height` be determined by the content here.

Reply:
I think we can also swap the `transform`, `left`, and `top` for `display: flex`, `justify-content: center`, and `align-items: center` in `blurBackground`. Also, we should use snake case for classes since `css` classes are case insensitive.